src/miner_mux_port.erl

```diff
@@ -0,0 +1,124 @@
+%%%-------------------------------------------------------------------
+%% @doc miner gwmp-mux service manager
+%% Start and monitor the port that runs the external rust-based semtech
+%% gwmp mux service
+%% @end
+%%%-------------------------------------------------------------------
+-module(miner_mux_port).
```

Comment:
Can we dedup this with the other port module any?

Reply:
like a port behaviour? are these likely to stick around long enough to abstract that commonality out or do we want to just maintain until we transition to lgws?